tap_oracle/__init__.py

```diff
@@ -344,6 +344,13 @@ def do_discovery(conn_config, filter_schemas):
         'is_view': True
      }
 
+   # If table_info is empty, send a SYMON error that no table was found with current input.
+   if not table_info:
+      cur.close()
+      connection.close()
+      # Raise database error because only database error messages are interpretted during discovery
+      raise cx_Oracle.DatabaseError('SYM-00001: No Table Found in the Database')
```

Comment:
what is cx_oracle? 
> only database error message are interpretted during discovery
can you link the doc that specifies this?  i just want to read more about it